lib/domains/navigation/hooks/useKeyboardNavigation.ts

```
@@ -0,0 +1,104 @@
+
+    const menuItems = getMenuItems(menuRef.current)
+    const currentIndex = menuItems.findIndex(item => item === document.activeElement)
+
+    switch (e.key) {
+      case KEYBOARD_KEYS.ARROW_DOWN:
+        e.preventDefault()
+        const nextIndex = getNextIndex(currentIndex, menuItems.length)
+        menuItems[nextIndex]?.focus()
+        break
+        
+      case KEYBOARD_KEYS.ARROW_UP:
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)